### ISSUE TITLE
Incorrect name

### ISSUE THREAD

**Issue by mgol** [dumconstantin/webpack-append#1]
"append" means to add something to the end, this module is prepending, not appending.

It so happens I was searching for an appending loader, I found this one and was surprised that it's doing sth different than its name suggests...

**Comment by Cleod9** [comment]
Agreed, I was just about to write the same issue 😄

**Comment by calumjames** [comment]
Yeah, I was almost put off using this before I saw on the NPM page that it did what I want because I thought it would append things instead of prepending things.
